### DISCUSSION TITLE
Generated classes properties setters not available

### DISCUSSION THREAD

**Discussion by emiliodel83** [ChilliCream/hotchocolate#3753] [Q&A]
Hello,
I have a question. When I download and generate code for a graphql schema I see that all interfaces and classes have a get but not a set. Also there is only one constructor and no parameter less constructor is available.
I have a page where I need to use the classes as models and edit them and save the changes.
Do you know if it is a framework limitation not having setters in properties or there is something I have to do.
Thanks!

**Comment by mbonacci** [comment]
Hi, I'm having the same problem and it seems to me like this is by design. I'm not sure what is the motivation for this, but it breaks a simple use case [emiliodel83](https://github.com/emiliodel83) described. Is there some other way to achieve this without creating your own classes and projecting query results?
